.github/PULL_REQUEST_TEMPLATE/manual_maintenance.md

```diff
@@ -192,6 +192,13 @@ check that all ssh-keys are set and erased from people that have left
 - [ ] After(!) the cypress tests have passed locally, execute the backend-e2e-tests `./gradlew dataland-e2etests:test`
 - [ ] Locally: Go to the swagger-UI, authorize, run a "GET" request to the companies endpoint and assure that your
   authorization has worked by assuring that you get a 200 response
+- [ ] It is assured that deploying this feature branch over the current main does not break anything
```

Comment:
What was the reasoning behind doing a migration test in every manual maintenance?

Reply:
This is something I proposed in a refinement meeting three weeks ago.  No one was against it in the refinement meeting.
That is why I just randomly chose this PR here to do it.

The reason is that we want to avoid that the version update of some library (especially Postgres DB is prone for this) breaks Dataland when re-deploy over the existing data on prod.

Reply:
Ok, fine for me. As i read the code i did not remember the refinement and was just wondering about it, but we should reinspect our complexity 1 for the MM. I feel that MM gets bigger and bigger and the 1 might be an underestimation.